cmd/antrea-agent/agent.go

```diff
@@ -145,13 +145,19 @@ func run(o *Options) error {
 	// notifying NetworkPolicyController to reconcile rules related to the
 	// updated Pods.
 	podUpdates := make(chan v1beta2.PodReference, 100)
+	// We set flow poll interval as the time interval for rule deletion in the async
+	// rule cache, which is implemented as the part of idAllocator. This is to preserve
+	// the rule info for mapping in the flow records of Flow Exporter even after
```

Comment:
the rule info for populating NetworkPolicy fields in the Flow Exporter even after rule deletion